package.json

```
@@ -145,5 +145,8 @@
     "vlq": "0.2.2",
     "vrsource-tslint-rules": "5.1.1",
     "webpack": "1.12.9"
+  },
+  "resolutions": {
+    "natives": "1.1.6"
```

I see that you documented the reason in the commit message. can you also add a brief description/reason into the package.json like this... thanks! (this will be helpful to know why it's here in the future and when it's ok to remove it)

```suggestion
    "//": "<reason why we need this override>"
    "natives": "1.1.6"
```